### ISSUE TITLE
[Bug] Bug title

### ISSUE THREAD

**Issue by nebeker** [ProtonMail/proton-mail-android#43]
**Describe the bug**
Some buttons an controls are not labelled for TalkBack, making the app very inaccessible.

**To Reproduce**
1. Turn on TalkBack
2. Open left side menu
3. Account expand switch is unlabelled
4. Notifications button is unlabelled - additionally, the only way to get back from the notification snooze dialog is to double tap outside of it and the back button doesn't work
5. Tap Compose message
6. +/- button for Cc. and Bcc. fields is unlabelled
7. Cc. and Bcc. edit boxes are unlabelled
8. Back out to Inbox and ope message
9. Start/unstar button is onlabelled
10. "Lock" icon is unlabelled

**Expected behavior**
Every button, switch or control should be labelled for TalkBack users, as per the mission statement: "our mission is to defend the right to privacy by making end-to-end encrypted email easy enough for anyone to use."

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: uses Google Play Servies
 - OS: Android 10
